broker/src/main/java/io/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -463,7 +463,7 @@ private void updatePositionOnSkipIfUpToDate(final long eventPosition) {
     }
 
     private void updateExporterLastExportedRecordPosition(final long eventPosition) {
-      state.setPosition(getId(), eventPosition);
+      state.setPositionIfGreater(getId(), eventPosition);
       metrics.setLastUpdatedExportedPosition(getId(), eventPosition);
       position = eventPosition;
```

Comment:
These should be updated only if `eventPosition` is greater than the current.

Reply:
I have totally overseen that thanks. This reminds me that the export director is currently really hard to test. Maybe I can change that.